### ISSUE TITLE
Thunderbird support

### ISSUE THREAD

**Issue by Felorek** [mkiol/GNotifier#9]
Hi would you add a support for thunderbird? It's working fine with firefox, but I think thunderbird needs this extension a little more.

**Comment by orschiro** [comment]
Any idea on how I can restrict notifications to be only shown for individual new messages but not the summation of all my apparently unread messages?

![screenshot](https://cloud.githubusercontent.com/assets/170947/5663416/eca91036-9746-11e4-866e-9dddf27f94ee.png)

**Comment by mkiol** [comment]
I am not sure if I fully understand you question. 

Notification is shown when FF detects new message. Content of notification is:
- sender and title of most recent new message in mail folder
- or "Number of unread messages: XX" if you set "Simple New mail notification" in settings 

What would you like to change?

**Comment by orschiro** [comment]
I don't want to use the `Simple new mail notification`. In the contrary, I only want to have shown the detailed one. Let me rephrase it.

This notification I want to have displayed:

![screenshot](https://cloud.githubusercontent.com/assets/170947/5695312/e158b496-999b-11e4-836e-4e7cda5c38b1.png)

This one not:

![](https://cloud.githubusercontent.com/assets/170947/5663416/eca91036-9746-11e4-866e-9dddf27f94ee.png)

**Comment by mkiol** [comment]
OK, I see. I will try to dig in this during weekend.

**Comment by v00d00** [comment]
It would also be great if GNotifier supported the "Server unavailable" message as that is "default" currently.

**Comment by mkiol** [comment]
@orschiro I've created separate issue #45 for problem you have reported.

**Comment by mkiol** [comment]
@v00d00 What is the easiest way to generate "Server unavailable" message? Honestly, I have never seen this notification on my TB.

**Comment by v00d00** [comment]
@mkiol Firewall your IMAP server :)

**Comment by zlamalp** [comment]
Hi, it seems that your extension somehow adds default action "open" to other notifications in TB. I use also SOGO extension which synchronize my contacts. On startup, it shows messages how many contacts were synchronized. There was never button before, just message.
 
![screen](https://cloud.githubusercontent.com/assets/1336607/9104478/4c37a4ac-3c0d-11e5-8983-d4d693165b78.png)

**Comment by mkiol** [comment]
@zlamalp 
What kind of notification server are you using? Xfce?

> There was never button before, just message.

What it means when you say "was"? Before installing latest version of GNotifier or before installing Gnotifier at all?

**Comment by zlamalp** [comment]
Hi @mkiol. Yes I'm using xfce (in xubuntu 15.04) and it was displaying plain message without button before installing Gnotifier at all (I tried 1.9.0 and 1.9.1-pre). When I uninstall Gnotifier, it behaves as before.

**Comment by mkiol** [comment]
@zlamalp Thanks for the clarification. One more question, so I understand that before installing GNotifier, SOGO add-on was sending "native" notifications (via xfce notification server) without "Open" button, right?

**Comment by zlamalp** [comment]
> One more question, so I understand that before installing GNotifier, SOGO add-on was sending "native" notifications (via xfce notification server) without "Open" button, right?

@mkiol Yes. It was showing notification without any button before I installed GNotifier on the same xfce environment.
